http-api/src/lib.rs

```diff
@@ -251,6 +260,34 @@ async fn readme_handler(
     Ok(warp::reply::json(&blob))
 }
 
+/// List all projects
+async fn project_root_handler(ctx: Context) -> Result<Json, Rejection> {
+    use radicle_daemon::git::identities::SomeIdentity;
+    #[derive(serde::Serialize)]
+    struct Project {
+        name: String,
+        urn: String,
+    }
+
+    let storage = ReadOnly::open(&ctx.paths).unwrap();
+    let projects = identities::any::list(&storage).unwrap()
+        .filter_map(|res| {
+            res.map(|id| match id {
+                SomeIdentity::Project(project) => {
+                    let rad_self = Reference::rad_self(Namespace::from(project.urn()), None);
+                    let name = (&project.payload().subject.name).to_string();
+                    let urn = Urn::try_from(rad_self).ok().unwrap().to_string()
+                        .split('/').next().unwrap().to_string();
```

Comment:
If you use the URN from the `link_identities::urn::Urn`, you can just call `urn.encode_id()`

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/75/commits/4e629abc369bc3d005f87bbd99a7dfb65987b450#diff-191ca8c6d151a1a4c5243319a8395c8a17c03c6e257bf4d796b004f19a0bf469L279-R282

Reply:
Btw, this is returning:
```
[{"name":"delete-this","urn":"hnrkyfoxs5xp45wg1p8q3dkh74qyebqaeunqo"}]
```
whereas api needs the full `/v1/projects/rad:git:hnrkyfoxs5xp45wg1p8q3dkh74qyebqaeunqo` version, which is what we were getting before.

Reply:
Oh you're right, my bad! Are we just trying to remove a path eg `/foo`? I don't think there should be any path returned :thinking: 

I think I would just use the `Serialize` instance of `Urn` and include it in `Project` as-is.